dnf5.spec

```diff
@@ -777,6 +777,21 @@ Package management service with a DBus interface.
 %{_mandir}/man8/dnf5daemon-server.8.*
 %{_mandir}/man8/dnf5daemon-dbus-api.8.*
 %endif
+
+
+# ========== dnf5daemon-server-polkit ==========
+
+%package -n dnf5daemon-server-polkit
+Summary:        Polkit rule to allow wheel group members install trusted packages
+License:        GPL-2.0-or-later
+Requires:       polkit
```

Comment:
You might also add `BuildArch: noarch` , due to the package containing only a text file, nothing built nor arch specific.

Reply:
That's a good point. Addressed by PR https://github.com/rpm-software-management/dnf5/pull/2283